src/js/pages/TeamHome.jsx

```diff
@@ -97,14 +101,16 @@ const TeamHome = ({ classes }) => {
               // showHeaderLabels={(teamMemberList && teamMemberList.length > 0)}
               showIcons={false}
             />
-            <TeamMemberList teamId={teamId} />
+            {/* PLEASE DO NOT REMOVE PASSED team */}
+            <TeamMemberList teamId={teamId} team={team} />
           </>
         )}
         <Button
           classes={{ root: classes.addTeamMemberButtonRoot }}
           color="primary"
           variant="outlined"
           onClick={addTeamMemberClick}
+          sx={{ marginTop: '30px' }}
```

Comment:
I'd like to keep all of the styles in one place, and move this "marginTop" style down to the 'addTeamMemberButtonRoot' definition near the bottom of this file.

Reply:
Good idea and No problem.  That seemed like a temporary layout